EN/asciidoc/src/04_solution_strategy.adoc

```diff
@@ -5,21 +5,21 @@
 [role="arc42help"]
 ****
 .Contents
-A short summary and explanation of the fundamental decisions and solution strategies, that shape the system's architecture. These include
+A short summary and explanation of the fundamental decisions and solution strategies, that shape system architecture. It includes
 
 * technology decisions
 * decisions about the top-level decomposition of the system, e.g. usage of an architectural pattern or design pattern
 * decisions on how to achieve key quality goals
 * relevant organizational decisions, e.g. selecting a development process or delegating certain tasks to third parties.
 
 .Motivation
-These decisions form the cornerstones for your architecture. They are the basis for many other detailed decisions or implementation rules.
+These decisions form the cornerstones for your architecture. They are the base for many other detailed decisions or implementation rules.
 
 .Form
-Keep the explanation of these key decisions short.
+Keep the explanations of key decisions short.
```

Comment:
such key decisions

Reply:
Yepp makes it more clear and does not sound unneeded.